Procfile

```diff
@@ -1 +1 @@
-web: grunt dist && node devserver.js
\ No newline at end of file
+web: tar -zcf dev/node_modules.tar.gz node_modules/ && grunt dist && node devserver.js
```

Comment:
The tar step is to insure the node_modules in the dev environment deploy quickly to heroku correct? Or is there more going on here?